packages/core/addon/components/navi-visualizations/table.js

```diff
@@ -262,6 +262,27 @@ export default Component.extend({
     });
   }),
 
+  /**
+   * @property {Boolean} isEditingMode
+   */
+  isEditingMode: computed('isEditing', function() {
+    return featureFlag('enableTableEditing') && get(this, 'isEditing');
+  }),
+
+  /**
+   * @property {Boolean} isVerticalCollectionEnabled
+   */
+  isVerticalCollectionEnabled: computed(function() {
+    return featureFlag('enableVerticalCollectionTableIterator');
+  }),
+
+  /**
+   * @property {String} tableRenderer
+   */
+  tableRenderer: computed('isVerticalCollectionEnabled', function() {
+    return `table-renderer${get(this, 'isVerticalCollectionEnabled') ? '-vertical-collection' : ''}`;
+  }),
```

Comment:
make sure to test print

Reply:
still having an issue here with a vertical scroller appearing on the pdf output

Reply:
Can we see a sample of the pdf?

Reply:
Okay I just made a pdf and it looks fine. (no better or worse than we have now) if you look at the print/ route for table now, it has a scrollbar too, it's probably fine.

Reply:
so that's the thing, with what we have now (feature flag is off) the route has a scrollbar but the pdf doesn't (try it with a report with ~500 rows)
however if you turn the flag on, the pdf does have a scrollbar, trying to see where's the difference